src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -572,12 +572,12 @@ Autoplay can be changed in YouTube settings:
     <string name="revanced_preference_screen_channel_bar_title">Channel bar</string>
     <string name="revanced_preference_screen_channel_bar_summary">Hide or show components of the channel bar under videos.</string>
 
-    <string name="revanced_hide_join_button_title">Hide join button</string>
+    <string name="revanced_hide_join_button_title">Hide Join button</string>
     <string name="revanced_hide_join_button_summary_on">Join button is hidden.</string>
     <string name="revanced_hide_join_button_summary_off">Join button is shown.</string>
-    <string name="revanced_hide_start_trial_button_title">Hide start trial button</string>
-    <string name="revanced_hide_start_trial_button_summary_on">Start trial button is hidden.</string>
-    <string name="revanced_hide_start_trial_button_summary_off">Start trial button is shown.</string>
+    <string name="revanced_hide_start_trial_button_title">"Hide 'Start trial' button"</string>
+    <string name="revanced_hide_start_trial_button_summary_on">"'Start trial' button is hidden."</string>
+    <string name="revanced_hide_start_trial_button_summary_off">"'Start trial' button is shown."</string>
```

Comment:
I wasn't able to find this button, but I assumed the button is labeled "Start trial" and therefore put it in quotes

Reply:
This button is only visible on channels that offer `Start trial`

If there is no channel offering `Start trial` in your country, try this:

1. Connect to UK VPN
2. Open the following video on unpatched YouTube:
https://youtu.be/FfpeFiMsHjM
3. You can see a `Start trial` button on the channel bar